libgadget/treewalk.c

```diff
@@ -321,14 +321,13 @@ treewalk_build_queue(TreeWalk * tw, int * active_set, const int size, int may_ha
     tw->WorkSet = mymalloc("ActiveQueue", tsize * sizeof(int) * tw->NThread);
```

Comment:
What about adding a comment like "Watch out: tw->WorkSet may change a few lines later due to the realloc"?

Reply:
Done!